### PR TITLE
Add feature request template

### LINKED ISSUE #316: Create template for issues

### PR DESCRIPTION
**Pull request by nabriis**
Closes #316

### CODE REVIEW

**Review by chaozg — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Include code example if possible
```
Comment (amal-ghamdi) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]:
```suggestion
**Is your feature request related to a problem? Please describe.**

A clear and concise description of what the problem is. Include code example if possible
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```python
+import cuqi
+
+```
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (amal-ghamdi) [inline .github/ISSUE_TEMPLATE/feature_request.md:18]:
```suggestion
**Describe the solution you'd like**

A clear and concise description of what you want to happen.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (amal-ghamdi) [inline .github/ISSUE_TEMPLATE/feature_request.md:22]:
```suggestion
**Describe alternatives you've considered**

A clear and concise description of any alternative solutions or features you've considered.
```

**Review by amal-ghamdi — APPROVED** [review]
Thank you @nabriis! LGTM, 

I just suggested some spacing after the title so the description does not appear at the same line as the title (at least according to the way I view in my VS). It also would be nice if we can empty this in all CUQI plugins at some point. 
I am approving, it is good to go from my side.

**Pull request merged**